src/WebJobs.Script/BindingExtensions/ExtensionsManager.cs

```diff
@@ -67,14 +66,14 @@ public async Task AddExtensions(params ExtensionPackageReference[] references)
             await SaveAndProcessProjectAsync(project);
         }
 
-        private async Task SaveAndProcessProjectAsync(ProjectRootElement project)
+        private async Task SaveAndProcessProjectAsync(XmlDocument project)
         {
             string baseFolder = Path.GetTempPath();
 
             var tempFolder = Path.Combine(baseFolder, "Functions", "Extensions", Guid.NewGuid().ToString());
             Directory.CreateDirectory(tempFolder);
 
-            File.WriteAllText(Path.Combine(tempFolder, ExtensionsProjectFileName), project.RawXml);
+            File.WriteAllText(Path.Combine(tempFolder, ExtensionsProjectFileName), project.InnerXml);
```

Comment:
I think this actually need to be
```suggestion
            File.WriteAllText(Path.Combine(tempFolder, ExtensionsProjectFileName), project.OuterXml);
```

Reply:
Yes, I agree...